src/main/webapp/user-page.html

```diff
@@ -21,23 +21,39 @@
     <meta charset="UTF-8">
     <link rel="stylesheet" href="/css/main.css">
     <link rel="stylesheet" href="/css/user-page.css">
+	<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBJ-A9BQNUd6cR6IgbOnZwr26_62tB6LRI"></script>
     <script src="/js/user-page-loader.js"></script>
   </head>
   <body onload="buildUI();">
     <nav>
       <ul id="navigation">
         <li><a href="/"><img src="images/home-logo.png" height="25px"></a></li>
+		<li ><a><button id="notifications-btn">Notifications</button></a></li>
         <li><a href="/">Public Feed</a></li>
         <li><a href="/community.html">Community</a></li>
         <li><a href="/charities.html">Charities</a></li>
       </ul>
     </nav>
+	
+	<div id="notification-container"></div>
+	
     <Label id="page-title" class="h1-label">User Page</Label>
 	
+    <div id="map-container">
+	  <div id="map"></div>
+      <form id="location-form" method="POST" action="/locations">
+	    <textarea name="longitude" placeholder="longitude" rows=1 required></textarea>
```

Comment:
do you mind swapping the order? All the location points are in the form (latitude, longitude) and so this might me confusing.

Reply:
Yes, Of course!